Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2323,7 +2325,7 @@ BYE BYE BORDERS
     .z-cse.z-cse-dark #center_panel-body .x-box-inner[id^=collectorform] input[type=text],
     .z-cse.z-cse-dark #class_center_panel > .x-toolbar .x-form-item,
     .z-cse.z-cse-dark #detailCardPanel > div > div > div table.x-hbox-form-item,
-    .z-cse.z-cse-dark #filterButton,
+    .z-cse.z-cse-dark #contentPaneContainer input[type=button], /*ZEN-30221*/
```

Comment:
According to discussion with @carlcamera rules in the list have to be alphabetized to an extent.